dockerfiles/platform-gnn-gpu/README.md

```diff
@@ -1,45 +1,100 @@
 # HPCC Systems Machine Learning GNN GPU Docker Images
 
-## Tensorflow 2 with GPU support
+## Tensorflow with GPU support
 
-The new Tensorflow 2 and Nvidia Cuda libraries are not always compatible. It will be ideal if we can use Docker image by published by Tensorflow (Docker Hub tensorflow/tensorflow <version>-gpu) but the base image is Ubuntu 18.04 and we use Ubuntu 20.04. So instead we create the images by addiing Tensorflow 2 and Nvida Cuda, etc libraries on the top of hpccystems/platform-core image.
+### Support Tensorflow on a GPU with Docker
+Docker is the easiest way to run TensorFlow on a GPU since the host machine only requires the NVIDIA® driver (the NVIDIA® CUDA® Toolkit is not required).
+When you create a AKS cluster with GPU support VM size NVIDIA driver is installed on the nodes.
```

Comment:
When you create an AKS cluster with GPU support a VM sized NVIDIA driver is installed on the nodes.

Reply:
Fixed